Harvest.Web/package-lock.json

```diff
@@ -1,5 +1,13 @@
 {
   "name": "harvest.web",
   "version": "1.0.0",
-  "lockfileVersion": 1
+  "lockfileVersion": 2,
```

Comment:
Not sure this really makes sense, Harvest.Web doesn't have packages and shouldn't need a packages section in the package-lock.json file.

Reply:
hmmm strange, I don't remember adding that but I will remove it